.pre-commit-config.yaml

```diff
@@ -29,27 +22,26 @@ repos:
         - python
         - file
         - python
-
-    - entry: isort
-      exclude: "^.*/js/.*$"
-      files: ""
-      id: isort
+    - entry: mypy
+      exclude: ^$
+      files: "{{ cookiecutter.package_slug }}/"
+      id: mypy
       language: python
-      name: isort
+      name: mypy
       pass_filenames: true
       stages:
         - commit
       types:
         - python
-
-    - entry: mypy
-      exclude: ^$
-      # note: it would be good to extend this to other apps as well
-      files: "{{ cookiecutter.package_slug }}/"
-      id: mypy
+    - id: ruff
+      name: ruff
+      entry: ruff
       language: python
-      name: mypy
       pass_filenames: true
+      exclude: |
```

Comment:
I guess you should use a different format here.
I found this: https://github.com/astral-sh/ruff/blob/main/.pre-commit-config.yaml#L53